javascript/javascript1/week1/Age-ify.js

```diff
@@ -0,0 +1,6 @@
+const yearOfBirth = 2004;
+const yearFuture = 2027;
+
+let age = yearFuture-yearOfBirth;
```

Comment:
This can be changed to `const` since we don't change the age value after initialization